### ISSUE TITLE
Can you indicate where the code generated by the random box is?

### ISSUE THREAD

**Issue by Rzx520** [scuwyh2000/RandBox#10]

**Comment by Rzx520** [comment]
class GaussianFourierProjection(nn.Module):
    """Gaussian random features for encoding time steps."""

    def __init__(self, embed_dim, scale=30.):
        super().__init__()
        # Randomly sample weights during initialization. These weights are fixed
        # during optimization and are not trainable.
        self.W = nn.Parameter(torch.randn(embed_dim // 2) * scale, requires_grad=False)

    def forward(self, x):
        x_proj = x[:, None] * self.W[None, :] * 2 * np.pi
        return torch.cat([torch.sin(x_proj), torch.cos(x_proj)], dim=-1)
**Is this it?**

**Comment by scuwyh2000** [comment]
> class GaussianFourierProjection(nn.Module): """Gaussian random features for encoding time steps."""
> 
> ```
> def __init__(self, embed_dim, scale=30.):
>     super().__init__()
>     # Randomly sample weights during initialization. These weights are fixed
>     # during optimization and are not trainable.
>     self.W = nn.Parameter(torch.randn(embed_dim // 2) * scale, requires_grad=False)
> 
> def forward(self, x):
>     x_proj = x[:, None] * self.W[None, :] * 2 * np.pi
>     return torch.cat([torch.sin(x_proj), torch.cos(x_proj)], dim=-1)
> ```
> 
> **Is this it?**

No, In the training stage, the random boxes were produced [here](https://github.com/scuwyh2000/RandBox/blob/main/randbox/detector.py#L328).

**Comment by Rzx520** [comment]
Where is the code generated using Gaussian functions reflected？

**Comment by scuwyh2000** [comment]
> Where is the code generated using Gaussian functions reflected？

torch.rand() samples data from a Gaussian distribution.

**Comment by Wang0203** [comment]
[box_placeholder](https://github.com/scuwyh2000/RandBox/blob/9b1f7d8e855d2fe9e512e8ac929d3a5f38ede652/randbox/detector.py#L326) What's this for?
